### ISSUE TITLE
Session Manager - SSH Tunnel - plink.exe timeout not working

### ISSUE THREAD

**Issue by jcguevara** [HeidiSQL/HeidiSQL#709]
## Steps to reproduce this issue

1. Try to connect to a remote DB over a SSH Tunnel and fails due slow network connection.
2. Open Session Manager and increase the SSH Tunnel - **plink.exe timeout** value.
3. Try to stablish again the connection. Still fails.

## Current behavior

I have faced this issue in a quite slow network connection, it doesn't matter how much I have increased this value, it never worked. I'm pretty sure this is a timeout problem because I have followed the same steps described before in DBeaver Community Edition and it work; I had the same issue and after increasing the timeout value on DBeaver SSH config. I was able to get connected to my DB.

## Expected behavior

After increasing the **plink.exe timeout** value, HeidiSQL needs to wait that specific time before giving up connecting to the DB. Would be also helpful to specify in the GUI if the timeout value to introduce is in seconds (s) or miliseconds (ms)

## Environment

* **HeidiSQL version:** 10.2.0.5616
  
* **Database system and version:** MySQL 5.7.20
  
* **Operating system:** Cent OS 7

**Comment by vuther** [comment]
Holy shit, I've been sitting here debugging this issue for the last 5 hours with 2 of my colleagues. What version of PuTTY are you on? You can check by running plink.exe -V. If you're on 0.71, try 0.70 with the increased timeout. It worked for me. However, it will increase the time it takes to connect. I'll be submitting a bug report shortly to get HeidiSQL working with 0.71 of PuTTy.

**Comment by ansgarbecker** [comment]
@jcguevara can you still reproduce that? I cannot, for some reason. If the first connect fails due to a short timeout, and I am increasing the timeout value, the second one succeeds.

**Comment by jcguevara** [comment]
@ansgarbecker I have found this bug due an connection problem that was solved by my internet provider a couple hours later, actually I haven't used this feature again since the connection is fast enought. As @vuther correctly pointed out I was using the version 0.71 of Putty. Back then when I was facing the issue I have tried several times to connect with the DB, increasing each time the timeout value without success.
